### PR TITLE
Create Member “felix-sosa/_index”

### PR DESCRIPTION
**Pull request by flxsosa**
Automatically generated by Netlify CMS

### CODE REVIEW

**Comment by flxsosa** [comment]
Seems legit to me.

File: content/authors/felix-sosa/_index.md [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+    - course: BSc Computer Science
+      institution: University of Central Florida
+      year: 2018
+    - course: SM Computer Science
+      institution: Harvard
+      year: ""
+    - course: PhD Cognitive Science
+      institution: Harvard
+user_groups:
+  - Graduate Students
```
Comment (jmuchovej) [inline content/authors/felix-sosa/_index.md:31]: Can you add "Researchers" here -- it's used on the backend.

Also, can you add "Graduate Student" or similar for your "role"?

File: content/authors/felix-sosa/_index.md [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+---
+I use the spaces of theoretical computer science, programming languages, and evolutionary theory to advance our understanding of the generative processes behind human intelligence. My current projects investigate the explanatory power programs have as a normative account of mental representation (e.g. if concepts are programs, how do people learn and use them?) and formalizing the development and evolution of our seemingly innate intuitions about the world (e.g. intuitive physics).
```
Comment (jmuchovej) [inline content/authors/felix-sosa/_index.md]: Is this intentionally duplicated from your `bio`? Take a look at:
- https://deploy-preview-9--cocodev.netlify.app/people/
- https://deploy-preview-9--cocodev.netlify.app/people/felix-sosa/

File: content/authors/felix-sosa/_index.md [hunk 3]
```diff
@@ -0,0 +1,32 @@
+---
+title: Felix Sosa
```
Comment (jmuchovej) [inline content/authors/felix-sosa/_index.md:2]: In the UI, there's a "Aliases" -- that's for when you have papers like... F. Sosa, or F. A. Sosa. I would recommend adding those as the site will highlight members of the lab on publications and the like.

**Review by jmuchovej — CHANGES_REQUESTED** [review]

File: content/authors/felix-sosa/_index.md (hunk 3 above)
Comment (flxsosa) [reply content/authors/felix-sosa/_index.md:2]: Done

File: content/authors/felix-sosa/_index.md (hunk 2 above)
Comment (flxsosa) [reply content/authors/felix-sosa/_index.md]: Fixed this too

**Review by jmuchovej — APPROVED** [review]

File: content/authors/felix-sosa/_index.md [hunk 4]
```diff
@@ -3,11 +3,7 @@ title: Felix Sosa
 avatar_filename: felix.jpg
 bio: I use the spaces of theoretical computer science, programming languages,
   and evolutionary theory to advance our understanding of the generative
-  processes behind human intelligence. My current projects investigate the
-  explanatory power programs have as a normative account of mental
-  representation (e.g. if concepts are programs, how do people learn and use
-  them?) and formalizing the development and evolution of our seemingly innate
-  intuitions about the world (e.g. intuitive physics).
+  processes behind human intelligence.
```
Comment (jmuchovej) [inline content/authors/felix-sosa/_index.md:11]: This looks like an incomplete thought?

**Review by jmuchovej — APPROVED** [review]

File: content/authors/felix-sosa/_index.md (hunk 4 above)
Comment (flxsosa) [reply content/authors/felix-sosa/_index.md:11]: Oh no I think something was clipped.

**Pull request merged**